apps/sertra/pages/server/[id].tsx

```diff
@@ -51,6 +52,10 @@ const ServerDetail: React.FC<{
 }> = ({ listing_data, mods_data }) => {
   return (
     <div className={styles.container}>
+      <HeadWrapper
+        title={`${listing_data.name} - V-Rising Servers | Thunderstore`}
```

Comment:
Could we read the game name from `listing_data`?

Reply:
Yes but it's lower case, I'll add it from there tho. Better to have dynamically.